WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/eclipse/logic/event/listeners/BreakpointListener.java

```diff
@@ -37,8 +37,8 @@
 	private final Map<Integer, Breakpoint> breakpoints;
 
 	/** Constructor. */
-	public BreakpointListener(EventManager eventManager) {
-		this.eventManager = eventManager;
+	public BreakpointListener(DebugEventManager debugEventManager) {
+		this.debugEventManager = debugEventManager;
 		this.breakpoints = new HashMap<>();
```

Comment:
I don't really see you point here, as there is currently only one manager that handles all debug events: DebugEventManager, which was first called EventManager. This class does not have a superclass and doesn't implement any interface, so I think there is no other type that we can use to parameterize this.